lib/foreman_tasks_core/continuous_output.rb

```diff
@@ -46,5 +46,14 @@ def self.format_output(message, type = 'debug', timestamp = Time.now.getlocal)
         'output' => message,
         'timestamp' => timestamp.to_f }
     end
+
+    def to_json_output
+      sort!
+      outputs_json = { 'outputs': [] }
+      raw_outputs.map do |output|
```

Comment:
`Enumerable.map` returns an array, using it for side effects isn't exactly best practice.

I'd go with either
```ruby
outputs = raw_outputs.map { |output| { type: output[:output_type], value: output[:message] } }
{ 'outputs': outputs }
```

or just swap `map` for `each` if you want to go the mutating way

Reply:
Thanks for the tip! I've updated that.